sphinx/source/docs/user_guide/interfaces.rst

```diff
@@ -19,22 +19,22 @@ browsers for presentation providing elegant, concise construction of novel
 graphics with high-performance interactivity over very large or streaming
 datasets in a quick and easy way.
 
-Offering both powerful and flexible features to enable very advanced
+To offer both powerful and flexible features, enabling advanced
 customizations on one hand and simplicity on the other, Bokeh exposes
 different interface levels to the users:
 
 *bokeh.models*
 --------------
 
-Bokeh is actually composed of two library components.
+Bokeh is composed of two library components.
```

Comment:
Also open to revision, but this was also intentional, to highlight the dichotomy itself, not just state it.